src/content_cao.cpp

```diff
@@ -690,6 +691,10 @@ v3f GenericCAO::getPosition()
 	}
 	return pos_translator.vect_show;
 }
+float GenericCAO::getYaw()
```

Comment:
Sorry, I am confused. Function is declaed and defined. Do you mean
inline float GenericCAO::getYaw() {return m_yaw;}
or something different?